cloud-run/README.md

```diff
@@ -274,3 +275,24 @@ The following example enables `canary` for  service. New revision of a service w
 Instead traffic percentage will be updated each `canary.interval` minutes using `canary.steps` values.
 On each step metrics for latest revision will be fetched and compared to `canary.thresholds`.
 If current metrics do not exceed thresholds, traffic will be increased otherwise servie will rollback to previous revision.
+
+#### Deployment with labels
+
+The following example shows how to add labels to the service. `component`, `iso-country`, `product` and `tenant-alias` are used in reports and must be on the format below. `iso-county` is the two lower case letters country code and should only be used in country specific services. `tenant-alias` is the alias for the tenant that can be found in iam, (plantagen, isrg) and should only be used in tenant specific services. `product` is the product the service belong to. `component` is the product component that the service belong to. It is possible to also add custom labels as seen below.
+
+Given the following `cloud-run.yaml`:
+```yaml
+name: my-service
+memory: 256Mi
+cpu: 1
+labels:
+  component: checkout-api
+  iso-country: se
+  product: hii-checkout
+  tenant-alias: plantagen
```

Comment:
Use more generic examples to not expose Extenda specifics or real tenant IDs.

Reply:
Fixed